### PR TITLE
rendering the correct layer 5 logo according to the initial theme

### LINKED ISSUE #1611: [Home] Dark mode compatibility for Layer5 logo
### Current Behavior

Screenshot 2024-01-22 at 6 12 44 PM

See layer5.io/brand for logos, if needed.

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by fardeenhasib**
**Description**

This PR fixes #1611 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by fardeenhasib** [comment]
@leecalcote could you review the changes ? Let me know if further changes are required.

**Comment by ritiksaxena124** [comment]
@fardeenhasib will you please remove the unnecessary changes created may be because of your prettier extension

**Comment by fardeenhasib** [comment]
> ritiksaxena124

got rid of the unnecessary changes. I had to set the autoformatting in VSCode to false.

File: _includes/home-page.html [hunk 1]
```diff
@@ -148,9 +148,19 @@ <h2 class="sub-text" style="padding-top:1rem; font-size: 1rem; font-weight: 500;
 				<h1 class="created-by">Created By</h1>
 				<div class="company-image">
 					<a href="https://layer5.io/" target="_blank">
-						<img id="layer5-logo" src="./assets/images/company-logo/layer5-no-trim.svg" alt="Layer5 logo"
+						<img id="layer5-logo" alt="Layer5 logo"
 							loading="lazy" />
 					</a>
+					<script>
+						const layer5_logo = document.getElementById("layer5-logo");
```
Comment (sudhanshutech) [inline _includes/home-page.html:155]: I think we already check this logic somewhere we don't have to describe everywhere this for the current theme. Use that logic . Check for meshery logo in header

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: _includes/home-page.html (hunk 1 above)
Comment (fardeenhasib) [reply _includes/home-page.html:155]: The line you've highlighted is not a logic. Its a variable assignment. What do you mean by "logic" here?

Comment (sudhanshutech) [reply _includes/home-page.html:155]: The logic you used to check for the current theme whether its light or dark , is already there you need to reuse that javascript

**Comment by saurabh100ni** [comment]
Hi @fardeenhasib  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

File: _includes/home-page.html (hunk 1 above)
Comment (fardeenhasib) [reply _includes/home-page.html]: Hi @sudhanshutech thanks for the clarification. Fixed resuing the script.

**Comment by leecalcote** [comment]
Great feedback, @sudhanshutech. Thank you. 👏

**Comment by sudhanshutech** [comment]
looks good👍, @Akanksha1212 remove merge conflict

**Review by sudhanshutech — DISMISSED** [review]

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
Remove this file `.jekyll-metadata` and its good to merge

**Comment by saurabh100ni** [comment]
@fardeenhasib, I see the empty check box in the PR description 👀, will you please mark it?

**Comment by leecalcote** [comment]
Other than the merge conflict, this looks good to me.

**Comment by sudhanshutech** [comment]
@fardeenhasib are you still there?

**Comment by fardeenhasib** [comment]
@sudhanshutech yes. Were you referring  to me when you said to remove the .jekyll-metdata file? I thought you were referring to someone else.

**Comment by sudhanshutech** [comment]
@fardeenhasib still merge conflicts

**Comment by sudhanshutech** [comment]
and you also rebase the branch, better to create a new pr

**Comment by fardeenhasib** [comment]
I tried rebasing the branch, but still the conflict persists. I guess, at the point, creating a new PR would be better altogether.

**Comment by leecalcote** [comment]
> I tried rebasing the branch, but still the conflict persists. I guess, at this point, creating a new PR would be better altogether.

Sounds like a good idea, @fardeenhasib

**Comment by leecalcote** [comment]
@fardeenhasib, let's roll...

**Pull request closed without merging**